metadata-transformer/src/main/resources/query.sparql

```diff
@@ -44,16 +43,16 @@ CONSTRUCT {
     BIND(BNODE("entityNode") AS ?entityNode)
     {
       SELECT ?title WHERE {
-        VALUES ?titleProperty { dcterms:title any23:citation_title }
-        <__URI__> ?titleProperty ?title  .
+        <__URI__> dcterms:title ?title  .
       } ORDER BY DESC(STRLEN(STR(?title))) LIMIT 1
     } UNION {
-      VALUES ?contributorProperty { dcterms:creator dcterms:contributor }
-      <__URI__> ?contributorProperty ?creator .
-
-      BIND(BNODE() AS ?contributorNode)
-      BIND(BNODE() AS ?identityNode)
-      FILTER (BOUND(?creator) && STRLEN(?creator)>0)
+        SELECT DISTINCT ?creator (SAMPLE(?cNode) AS ?contributorNode) (SAMPLE(?iNode) AS ?identityNode) WHERE {
```

Comment:
Removing duplicate creators. Previously, this was done by dint of RDF not duplicating triples (all creators were magically mapped to dcterms:creator), the change means that dcterms:contributor is kept intact and this must be handled. The ?creator must be distinct and the adjacent values are sampled from the solution.